src/app.py

```diff
@@ -13,34 +15,179 @@
 app = Dash(external_stylesheets=[dbc.themes.DARKLY])
 app.layout = dbc.Container([
     dcc.Store(id="filtered-data"),  # Used to store the data as it is filtered
+    
+    # First row containing only the title
     dbc.Row([
         dbc.Col([
-            html.H1("IMDB Dashboard"),
-            dbc.Checklist(
-                options=[
-                    {"label": genre, "value": genre} for genre in sorted(
-                        data.genres.unique().astype(str)
-                        ) if genre != "nan"
-                    ],
-                value=["Action", "Comedy", "Horror"],
-                id="genres-checklist",
-            ),
-        ]),
+            html.H1("IMDB Dashboard", style={'color': "#DBA506"}),
+        ])
+    ]),
+
+    # Second row containing filters towards left and charts toward right
+    dbc.Row([
         dbc.Col([
-            html.Iframe(
-                id='line',
-                style={'width': '100%', 'height': '400px'})
-            ]),
+            # First column containing filters separated by dividers
+            # Genre Checklist
+            html.Div([
+                html.H6(
+                    "Select Genres",
+                    style={'width': "150px", 'color': "#000000", 'font-weight': "bold", 'background': "#DBA506"}
+                    ),
+                dbc.Checklist(
+                    options=[
+                        {"label": genre, "value": genre} for genre in sorted(
+                            data.genres.unique().astype(str)
+                            ) if genre != "nan"
+                        ],
+                    value=["Action", "Comedy", "Horror"],
+                    id="genres-checklist",
+                    style={'width': "150px", 'height': "100%"}
+                    )
+                ]),
+            # Y-axis of line chart
             html.Div([
-                "Y-axis for line chart",
+                html.H6(
```

Comment:
This element and the dropdown below should be moved to under the line chart. That's outside the scope of this PR but I wanted to mention this so we have it on record.

Reply:
Nice point. Would make sense to have the chart-specific filter underneath the chart itself.